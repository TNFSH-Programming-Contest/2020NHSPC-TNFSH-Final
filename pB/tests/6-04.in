788
-924.23 73.851
-923.611 81.218
-922.934 88.58
-922.199 95.936
-921.405 103.286
-920.552 110.629
-919.64 117.966
-918.671 125.295
-917.642 132.616
-916.556 139.929
-915.411 147.232
-914.208 154.527
-912.947 161.811
-911.627 169.085
-910.25 176.349
-908.815 183.601
-907.322 190.842
-905.772 198.07
-904.164 205.286
-902.498 212.489
-900.775 219.678
-898.995 226.854
-897.157 234.015
-895.263 241.161
-893.312 248.291
-891.304 255.406
-889.239 262.505
-887.117 269.587
-884.94 276.652
-882.706 283.699
-880.415 290.728
-878.069 297.739
-875.667 304.731
-873.21 311.703
-870.697 318.656
-868.128 325.588
-865.505 332.5
-862.826 339.391
-860.092 346.26
-857.304 353.106
-854.461 359.931
-851.564 366.733
-848.613 373.511
-845.608 380.265
-842.549 386.996
-839.436 393.701
-836.271 400.382
-833.051 407.038
-829.78 413.667
-826.455 420.27
-823.077 426.846
-819.648 433.396
-816.166 439.917
-812.632 446.411
-809.047 452.876
-805.41 459.313
-801.723 465.72
-797.984 472.098
-794.194 478.446
-790.354 484.763
-786.463 491.049
-782.523 497.305
-778.533 503.528
-774.493 509.72
-770.404 515.879
-766.267 522.006
-762.08 528.099
-757.845 534.159
-753.562 540.184
-749.231 546.176
-744.852 552.132
-740.426 558.054
-735.953 563.94
-731.433 569.79
-726.866 575.604
-722.254 581.381
-717.595 587.122
-712.891 592.825
-708.141 598.49
-703.347 604.118
-698.507 609.706
-693.624 615.257
-688.696 620.768
-683.724 626.239
-678.709 631.671
-673.651 637.063
-668.55 642.414
-663.406 647.724
-658.221 652.993
-652.993 658.221
-647.724 663.406
-642.414 668.55
-637.063 673.651
-631.671 678.709
-626.239 683.724
-620.768 688.696
-615.257 693.624
-609.706 698.507
-604.118 703.347
-598.49 708.141
-592.825 712.891
-587.122 717.595
-581.381 722.254
-575.604 726.866
-569.79 731.433
-563.94 735.953
-558.054 740.426
-552.132 744.852
-546.176 749.231
-540.184 753.562
-534.159 757.845
-528.099 762.08
-522.006 766.267
-515.879 770.404
-509.72 774.493
-503.528 778.533
-497.305 782.523
-491.049 786.463
-484.763 790.354
-478.446 794.194
-472.098 797.984
-465.72 801.723
-459.313 805.41
-452.876 809.047
-446.411 812.632
-439.917 816.166
-433.396 819.648
-426.846 823.077
-420.27 826.455
-413.667 829.78
-407.038 833.051
-400.382 836.271
-393.701 839.436
-386.996 842.549
-380.265 845.608
-373.511 848.613
-366.733 851.564
-359.931 854.461
-353.106 857.304
-346.26 860.092
-339.391 862.826
-332.5 865.505
-325.588 868.128
-318.656 870.697
-311.703 873.21
-304.731 875.667
-297.739 878.069
-290.728 880.415
-283.699 882.706
-276.652 884.94
-269.587 887.117
-262.505 889.239
-255.406 891.304
-248.291 893.312
-241.161 895.263
-234.015 897.157
-226.854 898.995
-219.678 900.775
-212.489 902.498
-205.286 904.164
-198.07 905.772
-190.842 907.322
-183.601 908.815
-176.349 910.25
-169.085 911.627
-161.811 912.947
-154.527 914.208
-147.232 915.411
-139.929 916.556
-132.616 917.642
-125.295 918.671
-117.966 919.64
-110.629 920.552
-103.286 921.405
-95.936 922.199
-88.58 922.934
-81.218 923.611
-73.851 924.23
-66.479 924.789
-59.103 925.29
-51.724 925.732
-44.341 926.115
-36.955 926.439
-29.567 926.704
-22.177 926.91
-14.785 927.058
-7.393 927.146
0.0 927.175
7.393 927.146
14.785 927.058
22.177 926.91
29.567 926.704
36.955 926.439
44.341 926.115
51.724 925.732
59.103 925.29
66.479 924.789
73.851 924.23
81.218 923.611
88.58 922.934
95.936 922.199
103.286 921.405
110.629 920.552
117.966 919.64
125.295 918.671
132.616 917.642
139.929 916.556
147.232 915.411
154.527 914.208
161.811 912.947
169.085 911.627
176.349 910.25
183.601 908.815
190.842 907.322
198.07 905.772
205.286 904.164
212.489 902.498
219.678 900.775
226.854 898.995
234.015 897.157
241.161 895.263
248.291 893.312
255.406 891.304
262.505 889.239
269.587 887.117
276.652 884.94
283.699 882.706
290.728 880.415
297.739 878.069
304.731 875.667
311.703 873.21
318.656 870.697
325.588 868.128
332.5 865.505
339.391 862.826
346.26 860.092
353.106 857.304
359.931 854.461
366.733 851.564
373.511 848.613
380.265 845.608
386.996 842.549
393.701 839.436
400.382 836.271
407.038 833.051
413.667 829.78
420.27 826.455
426.846 823.077
433.396 819.648
439.917 816.166
446.411 812.632
452.876 809.047
459.313 805.41
465.72 801.723
472.098 797.984
478.446 794.194
484.763 790.354
491.049 786.463
497.305 782.523
503.528 778.533
509.72 774.493
515.879 770.404
522.006 766.267
528.099 762.08
534.159 757.845
540.184 753.562
546.176 749.231
552.132 744.852
558.054 740.426
563.94 735.953
569.79 731.433
575.604 726.866
581.381 722.254
587.122 717.595
592.825 712.891
598.49 708.141
604.118 703.347
609.706 698.507
615.257 693.624
620.768 688.696
626.239 683.724
631.671 678.709
637.063 673.651
642.414 668.55
647.724 663.406
652.993 658.221
658.221 652.993
663.406 647.724
668.55 642.414
673.651 637.063
678.709 631.671
683.724 626.239
688.696 620.768
693.624 615.257
698.507 609.706
703.347 604.118
708.141 598.49
712.891 592.825
717.595 587.122
722.254 581.381
726.866 575.604
731.433 569.79
735.953 563.94
740.426 558.054
744.852 552.132
749.231 546.176
753.562 540.184
757.845 534.159
762.08 528.099
766.267 522.006
770.404 515.879
774.493 509.72
778.533 503.528
782.523 497.305
786.463 491.049
790.354 484.763
794.194 478.446
797.984 472.098
801.723 465.72
805.41 459.313
809.047 452.876
812.632 446.411
816.166 439.917
819.648 433.396
823.077 426.846
826.455 420.27
829.78 413.667
833.051 407.038
836.271 400.382
839.436 393.701
842.549 386.996
845.608 380.265
848.613 373.511
851.564 366.733
854.461 359.931
857.304 353.106
860.092 346.26
862.826 339.391
865.505 332.5
868.128 325.588
870.697 318.656
873.21 311.703
875.667 304.731
878.069 297.739
880.415 290.728
882.706 283.699
884.94 276.652
887.117 269.587
889.239 262.505
891.304 255.406
893.312 248.291
895.263 241.161
897.157 234.015
898.995 226.854
900.775 219.678
902.498 212.489
904.164 205.286
905.772 198.07
907.322 190.842
908.815 183.601
910.25 176.349
911.627 169.085
912.947 161.811
914.208 154.527
915.411 147.232
916.556 139.929
917.642 132.616
918.671 125.295
919.64 117.966
920.552 110.629
921.405 103.286
922.199 95.936
922.934 88.58
923.611 81.218
924.23 73.851
924.789 66.479
925.29 59.103
925.732 51.724
926.115 44.341
926.439 36.955
926.704 29.567
926.91 22.177
927.058 14.785
927.146 7.393
927.175 0.0
927.146 -7.393
927.058 -14.785
926.91 -22.177
926.704 -29.567
926.439 -36.955
926.115 -44.341
925.732 -51.724
925.29 -59.103
924.789 -66.479
924.23 -73.851
923.611 -81.218
922.934 -88.58
922.199 -95.936
921.405 -103.286
920.552 -110.629
919.64 -117.966
918.671 -125.295
917.642 -132.616
916.556 -139.929
915.411 -147.232
914.208 -154.527
912.947 -161.811
911.627 -169.085
910.25 -176.349
908.815 -183.601
907.322 -190.842
905.772 -198.07
904.164 -205.286
902.498 -212.489
900.775 -219.678
898.995 -226.854
897.157 -234.015
895.263 -241.161
893.312 -248.291
891.304 -255.406
889.239 -262.505
887.117 -269.587
884.94 -276.652
882.706 -283.699
880.415 -290.728
878.069 -297.739
875.667 -304.731
873.21 -311.703
870.697 -318.656
868.128 -325.588
865.505 -332.5
862.826 -339.391
860.092 -346.26
857.304 -353.106
854.461 -359.931
851.564 -366.733
848.613 -373.511
845.608 -380.265
842.549 -386.996
839.436 -393.701
836.271 -400.382
833.051 -407.038
829.78 -413.667
826.455 -420.27
823.077 -426.846
819.648 -433.396
816.166 -439.917
812.632 -446.411
809.047 -452.876
805.41 -459.313
801.723 -465.72
797.984 -472.098
794.194 -478.446
790.354 -484.763
786.463 -491.049
782.523 -497.305
778.533 -503.528
774.493 -509.72
770.404 -515.879
766.267 -522.006
762.08 -528.099
757.845 -534.159
753.562 -540.184
749.231 -546.176
744.852 -552.132
740.426 -558.054
735.953 -563.94
731.433 -569.79
726.866 -575.604
722.254 -581.381
717.595 -587.122
712.891 -592.825
708.141 -598.49
703.347 -604.118
698.507 -609.706
693.624 -615.257
688.696 -620.768
683.724 -626.239
678.709 -631.671
673.651 -637.063
668.55 -642.414
663.406 -647.724
658.221 -652.993
652.993 -658.221
647.724 -663.406
642.414 -668.55
637.063 -673.651
631.671 -678.709
626.239 -683.724
620.768 -688.696
615.257 -693.624
609.706 -698.507
604.118 -703.347
598.49 -708.141
592.825 -712.891
587.122 -717.595
581.381 -722.254
575.604 -726.866
569.79 -731.433
563.94 -735.953
558.054 -740.426
552.132 -744.852
546.176 -749.231
540.184 -753.562
534.159 -757.845
528.099 -762.08
522.006 -766.267
515.879 -770.404
509.72 -774.493
503.528 -778.533
497.305 -782.523
491.049 -786.463
484.763 -790.354
478.446 -794.194
472.098 -797.984
465.72 -801.723
459.313 -805.41
452.876 -809.047
446.411 -812.632
439.917 -816.166
433.396 -819.648
426.846 -823.077
420.27 -826.455
413.667 -829.78
407.038 -833.051
400.382 -836.271
393.701 -839.436
386.996 -842.549
380.265 -845.608
373.511 -848.613
366.733 -851.564
359.931 -854.461
353.106 -857.304
346.26 -860.092
339.391 -862.826
332.5 -865.505
325.588 -868.128
318.656 -870.697
311.703 -873.21
304.731 -875.667
297.739 -878.069
290.728 -880.415
283.699 -882.706
276.652 -884.94
269.587 -887.117
262.505 -889.239
255.406 -891.304
248.291 -893.312
241.161 -895.263
234.015 -897.157
226.854 -898.995
219.678 -900.775
212.489 -902.498
205.286 -904.164
198.07 -905.772
190.842 -907.322
183.601 -908.815
176.349 -910.25
169.085 -911.627
161.811 -912.947
154.527 -914.208
147.232 -915.411
139.929 -916.556
132.616 -917.642
125.295 -918.671
117.966 -919.64
110.629 -920.552
103.286 -921.405
95.936 -922.199
88.58 -922.934
81.218 -923.611
73.851 -924.23
66.479 -924.789
59.103 -925.29
51.724 -925.732
44.341 -926.115
36.955 -926.439
29.567 -926.704
22.177 -926.91
14.785 -927.058
7.393 -927.146
-0.0 -927.175
-7.393 -927.146
-14.785 -927.058
-22.177 -926.91
-29.567 -926.704
-36.955 -926.439
-44.341 -926.115
-51.724 -925.732
-59.103 -925.29
-66.479 -924.789
-73.851 -924.23
-81.218 -923.611
-88.58 -922.934
-95.936 -922.199
-103.286 -921.405
-110.629 -920.552
-117.966 -919.64
-125.295 -918.671
-132.616 -917.642
-139.929 -916.556
-147.232 -915.411
-154.527 -914.208
-161.811 -912.947
-169.085 -911.627
-176.349 -910.25
-183.601 -908.815
-190.842 -907.322
-198.07 -905.772
-205.286 -904.164
-212.489 -902.498
-219.678 -900.775
-226.854 -898.995
-234.015 -897.157
-241.161 -895.263
-248.291 -893.312
-255.406 -891.304
-262.505 -889.239
-269.587 -887.117
-276.652 -884.94
-283.699 -882.706
-290.728 -880.415
-297.739 -878.069
-304.731 -875.667
-311.703 -873.21
-318.656 -870.697
-325.588 -868.128
-332.5 -865.505
-339.391 -862.826
-346.26 -860.092
-353.106 -857.304
-359.931 -854.461
-366.733 -851.564
-373.511 -848.613
-380.265 -845.608
-386.996 -842.549
-393.701 -839.436
-400.382 -836.271
-407.038 -833.051
-413.667 -829.78
-420.27 -826.455
-426.846 -823.077
-433.396 -819.648
-439.917 -816.166
-446.411 -812.632
-452.876 -809.047
-459.313 -805.41
-465.72 -801.723
-472.098 -797.984
-478.446 -794.194
-484.763 -790.354
-491.049 -786.463
-497.305 -782.523
-503.528 -778.533
-509.72 -774.493
-515.879 -770.404
-522.006 -766.267
-528.099 -762.08
-534.159 -757.845
-540.184 -753.562
-546.176 -749.231
-552.132 -744.852
-558.054 -740.426
-563.94 -735.953
-569.79 -731.433
-575.604 -726.866
-581.381 -722.254
-587.122 -717.595
-592.825 -712.891
-598.49 -708.141
-604.118 -703.347
-609.706 -698.507
-615.257 -693.624
-620.768 -688.696
-626.239 -683.724
-631.671 -678.709
-637.063 -673.651
-642.414 -668.55
-647.724 -663.406
-652.993 -658.221
-658.221 -652.993
-663.406 -647.724
-668.55 -642.414
-673.651 -637.063
-678.709 -631.671
-683.724 -626.239
-688.696 -620.768
-693.624 -615.257
-698.507 -609.706
-703.347 -604.118
-708.141 -598.49
-712.891 -592.825
-717.595 -587.122
-722.254 -581.381
-726.866 -575.604
-731.433 -569.79
-735.953 -563.94
-740.426 -558.054
-744.852 -552.132
-749.231 -546.176
-753.562 -540.184
-757.845 -534.159
-762.08 -528.099
-766.267 -522.006
-770.404 -515.879
-774.493 -509.72
-778.533 -503.528
-782.523 -497.305
-786.463 -491.049
-790.354 -484.763
-794.194 -478.446
-797.984 -472.098
-801.723 -465.72
-805.41 -459.313
-809.047 -452.876
-812.632 -446.411
-816.166 -439.917
-819.648 -433.396
-823.077 -426.846
-826.455 -420.27
-829.78 -413.667
-833.051 -407.038
-836.271 -400.382
-839.436 -393.701
-842.549 -386.996
-845.608 -380.265
-848.613 -373.511
-851.564 -366.733
-854.461 -359.931
-857.304 -353.106
-860.092 -346.26
-862.826 -339.391
-865.505 -332.5
-868.128 -325.588
-870.697 -318.656
-873.21 -311.703
-875.667 -304.731
-878.069 -297.739
-880.415 -290.728
-882.706 -283.699
-884.94 -276.652
-887.117 -269.587
-889.239 -262.505
-891.304 -255.406
-893.312 -248.291
-895.263 -241.161
-897.157 -234.015
-898.995 -226.854
-900.775 -219.678
-902.498 -212.489
-904.164 -205.286
-905.772 -198.07
-907.322 -190.842
-908.815 -183.601
-910.25 -176.349
-911.627 -169.085
-912.947 -161.811
-914.208 -154.527
-915.411 -147.232
-916.556 -139.929
-917.642 -132.616
-918.671 -125.295
-919.64 -117.966
-920.552 -110.629
-921.405 -103.286
-922.199 -95.936
-922.934 -88.58
-923.611 -81.218
-924.23 -73.851
-924.789 -66.479
-925.29 -59.103
-925.732 -51.724
-926.115 -44.341
-926.439 -36.955
-926.704 -29.567
-926.91 -22.177
-927.058 -14.785
-927.146 -7.393
-927.175 0.0
-927.146 7.393
-927.058 14.785
-926.91 22.177
-926.704 29.567
-926.439 36.955
-926.115 44.341
-925.732 51.724
-925.29 59.103
-924.789 66.479
-350.421 890.869
