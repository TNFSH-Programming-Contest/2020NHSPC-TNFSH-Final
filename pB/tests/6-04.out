11153843.571